### ISSUE TITLE
Which parameters need to be modified if I use my own datasat?

### ISSUE THREAD

**Issue by TGISer** [zhangweichen2006/iCAN#2]
soure domain：21class，Each class has 800 images.
soure domain：21class，Each class has 100 images.

I modified the following variables：
NUM_CLASS = 21
data direction

Do I need to change other parameters, like：
PRETRAIN_SAMPLE = 60000
TRAIN_SAMPLE = 240000

Thank you for your help!

**Comment by zhangweichen2006** [comment]
dsets['source'] = datasets.ImageFolder(data_dir+source_set+'/images', data_transforms['source'])
dsets['target'] = datasets.ImageFolder(data_dir+target_set+'/images', data_transforms['target'])
dsets['pseudo_source'] = datasets.ImageFolder(data_dir+source_set+'/images', data_transforms['source'])
dsets['pseudo'] = []
dsets['test'] = datasets.ImageFolder(data_dir+target_set+'/images', data_transforms['test'])

You can change these above few lines, data_dir is the dataset directory, source_set and target_set(test_set) is the name, which is both String format(e.g."...").

**Comment by TGISer** [comment]
Maybe I didn't express it accurately enough, I've already changed the dataset directory(windows) like：
data_dir = "datasat\\"
dsets = {}
dsets['source'] = datasets.ImageFolder(data_dir+source_set, data_transforms['source'])
dsets['target'] = datasets.ImageFolder(data_dir+target_set, data_transforms['target'])
dsets['pseudo_source'] = datasets.ImageFolder(data_dir+source_set, data_transforms['source'])
dsets['pseudo'] = []
dsets['test'] = datasets.ImageFolder(data_dir+target_set, data_transforms['test'])

What I want to know is whether other parameters will affect the final accuracy, like PRETRAIN_SAMPLE、TRAIN_SAMPLE .

**Comment by zhangweichen2006** [comment]
Oh, this is hyper-parameter which needs to be tuned. You can try this one first, and find others if this setting is overfitting much.

**Comment by TGISer** [comment]
Thank you!
